src-docs/src/views/loading/loading_example.js

```diff
@@ -50,6 +55,25 @@ export const LoadingExample = {
     </OuiText>
   ),
   sections: [
+    {
+      title: 'Dashboards',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: loadingDashboardsSource,
+        },
+      ],
+      text: (
+        <p>
+          The <strong>OuiLoadingDashboards</strong> loader is great for full
+          page or Dashboards product loading screens.
```

Comment:
@ashwin-pc, please check this text, maybe a different one should be written here?

Reply:
@SergeyMyssak this text guidance on the use case for OpenSearch Dashboards loader still applies - thanks for checking !